var/spack/repos/builtin/packages/qmd-progress/package.py

```diff
@@ -47,6 +48,8 @@ class QmdProgress(CMakePackage):
     def cmake_args(self):
         spec = self.spec
         args = ['-DCMAKE_Fortran_FLAGS=-ffree-line-length-none']
+        if '+shared' in spec:
+            args.append('-DBUILD_SHARED_LIBS=ON')
```

Comment:
Please make this add `-DBUILD_SHARED_LIBS=OFF` in the case where `~shared` in spec, as you did for bml.

Reply:
`-DBUILD_SHARED_LIBS=OFF` is the default, but sure.